doozer/doozerlib/distgit.py

```diff
@@ -1611,7 +1627,7 @@ def _rebase_from_directives(self, dfp):
         for i, image in enumerate(parent_images):
             # Does this image inherit from an image defined in a different group member distgit?
             if image.member is not Missing:
-                mapped_images.append(self._mapped_image_from_member(image, original_parents, count))
+                mapped_images.append(self._mapped_image_from_member(image, original_parents[i], dfp))
```

Comment:
we can now delete `count`?

Reply:
Done in ea500e9de3b27acbc42e98c576995a353a09c1c8